TLM/TLM/Manager/Impl/ExtVehicleManager.cs

```diff
@@ -1041,6 +1041,54 @@ public class ExtVehicleManager
             }
         }
 
+        /// <summary>
+        /// Converts game VehicleInfo.VehicleType to closest TMPE.API.Traffic.Enums.ExtVehicleType
+        /// </summary>
+        /// <param name="vehicleType"></param>
+        /// <returns></returns>
+        public static ExtVehicleType ConvertToExtVehicleType(VehicleInfo.VehicleType vehicleType) {
```

Comment:
should be a train of if-elseif, not plain if if if? It checks entire set of ifs every time. If the if priority matters, then invert the order of ifs and connect them with else-if chain?